<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_Chunk">
     <property name="title">
      <string>Import Chunk</string>
     </property>
     <addaction name="actionSelectChunk"/>
     <addaction name="actionLoad_Chunk"/>
    </widget>
    <addaction name="menuImport_Chunk"/>
    <addaction name="actionClear_Chunks"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuNavigation_Type">
     <property name="title">
      <string>Navigation Type</string>
     </property>
     <addaction name="actionRotate"/>
     <addaction name="actionMove"/>
    </widget>
    <widget class="QMenu" name="menuRender_Mode">
     <property name="title">
      <string>Render Mode</string>
     </property>
     <widget class="QMenu" name="menuRender_Caves">
      <property name="title">
       <string>Render Caves</string>
      </property>
      <addaction name="actionRender_Caves"/>
      <addaction name="actionRenderCavesLightmap"/>
      <addaction name="actionRenderCavesTextured"/>
     </widget>
     <widget class="QMenu" name="menuRender_All">
      <property name="title">
       <string>Render All</string>
      </property>
      <addaction name="actionRender_All"/>
      <addaction name="actionRenderAllTextured"/>
     </widget>
     <widget class="QMenu" name="menuRender_Top">
      <property name="title">
       <string>Render Top</string>
      </property>
      <addaction name="actionRender_Top"/>
      <addaction name="actionRenderTopTextured"/>
     </widget>
     <addaction name="menuRender_All"/>
     <addaction name="menuRender_Top"/>
     <addaction name="menuRender_Caves"/>
    </widget>
    <addaction name="menuRender_Mode"/>
    <addaction name="menuNavigation_Type"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectChunk"/>
   <addaction name="actionLoad_Chunk"/>
   <addaction name="separator"/>
   <addaction name="actionClear_Chunks"/>
   <addaction name="separator"/>
   <addaction name="actionRotate"/>
   <addaction name="actionMove"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
   <addaction name="separator"/>
   <addaction name="actionTopDown"/>
   <addaction name="actionFirstPerson"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Chunk">
   <property name="text">
    <string>Load Chunk</string>
   </property>
  </action>
  <action name="actionClear_Chunks">
   <property name="text">
    <string>Clear Chunks</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRender_All">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionRender_Top">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionRender_Caves">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate Mode</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionSelectChunk">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Select Chunk</string>
   </property>
  </action>
  <action name="actionFirstPerson">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>First Person</string>
   </property>
  </action>
  <action name="actionTopDown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Top Down</string>
   </property>
  </action>
  <action name="actionRenderCavesLightmap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lightmap</string>
   </property>
  </action>
  <action name="actionRenderAllTextured">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Textured</string>
   </property>
  </action>
  <action name="actionRenderTopTextured">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Textured</string>
   </property>
  </action>
  <action name="actionRenderCavesTextured">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Textured</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
